confgenerator/confgenerator.go

```diff
@@ -74,9 +74,16 @@ func (uc *UnifiedConfig) GenerateOtelConfig(hostInfo *host.InfoStat) (string, er
 				"user_agent": userAgent,
 				"metric": map[string]interface{}{
 					// Receivers are responsible for sending fully-qualified metric names.
-					// NB: If a receiver fails to send a full URL, OT will add the prefix `custom.googleapis.com/opencensus/`.
+					// NB: If a receiver fails to send a full URL, OT will add the prefix `workload.googleapis.com/{metric_name}`.
 					// TODO(b/197129428): Write a test to make sure this doesn't happen.
 					"prefix": "",
+					// OT calls CreateMetricDescriptor by default. Skip because we want
+					// descriptors to be created implicitly with new time series.
+					"skip_create_descriptor": true,
+					// Omit instrumentation labels, which break agent metrics.
+					"instrumentation_library_labels": false,
```

Comment:
We will need to sync with @dashpole on this when we start the task on adding instrumentation_library_labels